db/schema.rb

```diff
@@ -11,15 +11,35 @@
 #
```

Comment:
Any idea why `rake db:migrate` is requiring so much change to the schema? Is it due to upgrading the versions of the `rails` or `pg` gems?